frontend/vue/components/Syllabus/SyllabusForm.vue

```diff
@@ -90,36 +100,81 @@ export default defineComponent({
           location: 'create-syllabus'
         }
       },
-      modules: [{
-        id: 1
-      }],
-      modulesCounter: 1
+      sections: [] as Section[],
+      syllabus: {
+        name: 'PHYS 332: Quantum Mechanics II (Spring, 2022)',
+        instructor: 'instructor name',
+        location: 'Madrid',
+        institution: 'UCM',
+        officeHours: '10:00 to 13:00',
+        classHours: '15:00 to 18:00',
+        email: 'none@none.never',
+        courseList: [
+          {
+            title: 'Week 1',
+            description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin vehicula tellus non ligula hendrerit interdum. Suspendisse sit amet erat vitae urna mattis sodales. Nullam consequat sagittis tellus. In et justo ex. Suspendisse tempor auctor blandit. Sed vel est eu felis vehicula varius id non ante. Morbi lacinia dolor ac nunc malesuada, dictum imperdiet ligula pellentesque.',
+            unitList: [
+              {
```

Comment:
Similar to what I mention previously. The backend initially will return an array of UUIDs like:

```js
['bdb1d662-e0f6-428a-8830-befe6b47f320', '7c765036-aebc-11ec-b909-0242ac120002']
```